core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/git/GitHandler.scala

```diff
@@ -75,20 +77,20 @@ object GitHandler {
     private val completion: AtomicInteger = new AtomicInteger()
     private val task: AtomicReference[String] = new AtomicReference()
 
+    private val titlePercentArray: mutable.ArrayBuffer[(String, Int)] = mutable.ArrayBuffer.empty
```

Comment:
Any risk of concurrent writes/race conditions here?

Reply:
I checked the logs and it doesn't look like our tests bring such risk.